### PR TITLE
hot-fix/missing ecrypt/decrypt function in web

### PR DESCRIPTION
**Pull request by sherlockvn**
### modifying in SDK:

https://github.com/linagora/matrix-dart-sdk/pull/6/

https://github.com/linagora/twake-on-matrix/assets/43041967/75651e36-61c1-43bf-ba70-90d13d430b5d

https://github.com/linagora/twake-on-matrix/assets/43041967/f6512c3a-5fc8-4bf4-9ea2-e7cdfdc631e8

https://github.com/linagora/twake-on-matrix/assets/43041967/b77975e4-e66a-40a1-9057-54308b9b7378

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -1445,8 +1445,8 @@ packages:
     dependency: "direct main"
     description:
       path: "."
-      ref: improve-matrix-file
-      resolved-ref: b8762469be46d3ec6ee782ccdc035a597202c2f4
+      ref: "hot-fix/failed-to-build-web"
```
Comment (drminh2807) [inline pubspec.lock:1448]: you must add the PR of dart sdk to the desc. So you can update this lock after the dependent PR merged

Comment (sherlockvn) [reply pubspec.lock:1448]: What do you mean ? @@

**Comment by hoangdat** [comment]
please update the new merged SDK

**Comment by sherlockvn** [comment]
> please update the new merged SDK

done

**Review by hoangdat — APPROVED** [review]

**Pull request merged**